sdk/analytics/src/main/java/com/klaviyo/analytics/networking/KlaviyoApiClient.kt

```diff
@@ -37,8 +56,11 @@ internal object KlaviyoApiClient : ApiClient {
 
         // Stop the background batching job while offline
         Registry.networkMonitor.onNetworkChange { isOnline ->
-            if (isOnline) startBatch(true)
-            else stopBatch()
+            if (isOnline) {
```

Comment:
Did we change the formmatter?

Reply:
I'm not sure why that happened, the pre-commit formatter is pretty lax, I might have done this myself absentmindedly.